app/schema.py

```diff
@@ -121,13 +124,23 @@ class MeetingUpdate(BaseModel):
     meeting_missed: Optional[Literal['Missed', 'Attended']]
 
 
+class Resource(BaseModel):
+    name: constr(max_length=255)
+    item_id: constr(max_length=255)
+
+
 class Feedback(BaseModel):
-    ticket_id: constr(max_length=255)
+    text: constr(max_length=255)
+    ticket_id: constr(max_length=16)
     mentee_id: constr(max_length=255)
     mentor_id: constr(max_length=255)
-    feedback: Optional[constr(max_length=2000)]
+    Datetime: datetime
+    vader_score: Optional[constr(max_length=255)]
 
+    def __init__(self, data) -> None:
+        super().__init__(data)
```

Comment:
I don't think this is going to do what you intended. Let's talk about it in Office Hours